src/Repository/TimesheetInvoiceItemRepository.php

```diff
@@ -35,6 +36,27 @@ public function getInvoiceItemsForQuery(InvoiceQuery $query): iterable
         return $this->repository->getTimesheetsForQuery($query, true);
     }
 
+    /**
+     * @param Invoice $invoice
+     * @return InvoiceItemInterface[]
+     */
+    public function getInvoiceItemsByInvoice(Invoice $invoice): iterable
+    {
+        return this->repository->getTimesheetsByInvoice(invoice, true);
```

Comment:
Did you test that code?

Reply:
Not yet. I planned to do that later today, if i find the time. Thanks for looking over it

Reply:
Okay, as I receive an email on every action (commit ...) can you send me a ping once you are ready? 
Just so I know that I can start reviewing instead.